.gitignore

```diff
@@ -99,3 +99,5 @@ saml_*.key
 yarn-debug.log*
 yarn-error.log*
 .yarn-integrity
+
+11892Notes.txt
```

Comment:
I'm guessing this is not intended to be added to the PR?

Reply:
It wasn't, thanks for catching that.  I just pushed a fix.